lms/djangoapps/courseware/management/commands/export_course.py

```diff
@@ -37,7 +37,7 @@ def handle(self, *args, **options):
 
         results = self._get_results(filename) if pipe_results else None
 
-        return results
+        self.stdout.write(results, ending="")
```

Comment:
After a brief investigation the print statement adds a space if the statement is not on a newline and a carriage return to the end of the statement. Using stdout.write will simply put the output to the buffer without any modifications.